cupy/cuda/runtime.pxd

```diff
@@ -249,6 +250,13 @@ cpdef enum:
     cudaChannelFormatKindFloat = 2
     cudaChannelFormatKindNone = 3
 
+    # CUDA array flags
+    cudaArrayDefault = 0
+    # cudaArrayLayered = 1
+    cudaArraySurfaceLoadStore = 2
```

Comment:
`cuda.runtime.cudaArrayDefault` was actually referred in `CUDAarray`'s docstring but we never added it, plus `cudaArraySurfaceLoadStore` is needed for surface memory, so I add them here.